### PR TITLE
Update GPC copy

### PR DESCRIPTION
**Pull request by cmonfortep**
Task/Issue URL: https://app.asana.com/0/488551667048375/1199892618151767/f
Tech Design URL: 
CC: https://app.asana.com/0/1142021229838617/1200212320487140/f

**Description**:
We are updating GPC copy based on the conversations in https://app.asana.com/0/1142021229838617/1200212320487140/f

**Steps to test this PR**:
1. Go to settings
1. Click on Global Privacy Control
2. Ensure new copy is displayed
3. Learn more link is clickable and navigates to learn more site

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

**Comment by cmonfortep** [comment]
Do not merge: Translations missing. We will open a product review and once gets approved we will request translations for the new copy.

File: app/src/main/res/values/strings.xml
Comment (CDRussell) [inline app/src/main/res/values/strings.xml:220]: Why have these been moved out of `untranslated`? They don't appear to be translated.

**Review by CDRussell — APPROVED** [review]

File: app/src/main/res/values/strings.xml
Comment (cmonfortep) [reply app/src/main/res/values/strings.xml:220]: Yes, I included them as part of the translation request, they will be translated before merging this PR.

**Pull request merged**
